README.md

```diff
@@ -223,13 +218,13 @@ async fn main() {
     .await;
 }
 
-async fn start(bot: Bot, msg: Message, dialogue: MyDialogue) -> HandlerResult {
+async fn start(bot: Bot, dialogue: MyDialogue, msg: Message) -> HandlerResult {
```

Comment:
If we want to be as consistent as possible, `Message` should go before `MyDialogue`

Reply:
Why? Dialogue can change less frequently than a message.

Reply:
Because dialogue in a sense is derived from the message (its chat id). Although if you disagree, leave as is, it's fine.

Reply:
I think derivability is more of an implementation detail since a user doesn't see that dialogue is derived from a message. Also, in future, it can change, e.g. derive a dialogue from something other than that `msg`.